src/native/corehost/corehost.cpp

```diff
@@ -312,6 +312,12 @@ int __cdecl wmain(const int argc, const pal::char_t* argv[])
 int main(const int argc, const pal::char_t* argv[])
 #endif
 {
+// Disable Swift backtrace on macOS to avoid conflicts with the host's signal handlers.
+// see https://github.com/dotnet/runtime/issues/118823
+#if defined(TARGET_OSX)
+    ::setenv("SWIFT_BACKTRACE", "enable=no", 0);
```

Comment:
Should we override it if the user set it explicitly? (I don't have strong opinion about it since the official builds of the dotnet host are built with hardened runtime and it would have no effect anyway.)

Reply:
Ah, I missed the resolved comment from @vcsjones, so ignore this.